CONTRIBUTING.md

```diff
@@ -6,82 +6,66 @@ worth knowing.
 see in terms of code formatting, so don't ignore him.
 
-## Gulp
-
-We compress/uglify our CSS and JS files using [Gulp](http://gulpjs.com/)
-via our very own [`product-taskmaster`](https://github.com/moderntribe/product-taskmaster)
-repo - which is a collection of gulp tasks. Here's how you get rolling
-with that.
+## Prepping the repository for usage/development
 
 ### Prerequisites
 
-#### get the submodules
-
-We rely on a few submodules, you'll need those to continue.
-
-```
-git submodule update --init --recursive
-```
+* [composer](https://getcomposer.org/download/)
+* [node](https://nodejs.org/download/)
+* [nvm](https://github.com/nvm-sh/nvm)
 
-#### Install Node.js
+### Build steps
 
-If you don't already have Node.js installed, please do so first:
+#### Get the submodules
 
-[Download Node.js](http://nodejs.org/download/)
-
-#### Install Gulp
-
-The only requirement for Gulp is the Gulp CLI (Command Line Utility). If
-you don't already have that installed, install it globally. You can do
-that with the npm `-g` flag.
+We rely on a few submodules, you'll need those to continue.
 
 ```
-npm install -g gulp-cli
+git submodule update --init --recursive
 ```
 
-#### Install all the NPM packages
+#### Install composer packages
 
-With all of those in place, run:
+_Note: you'll need to do this for the common directory and the root directory._
 
 ```
-npm install
-```
-
-If you run into any issues with some of the tasks down the road, run npm
-rebuild and try again.
+cd common
+composer install
 
+cd ..
+composer install
 ```
-npm rebuild
-```
-
-#### Install composer
 
-We use composer for some critical packages. Please [install composer](https://getcomposer.org/download/) (if you do not already have it).
+#### Install and build all the npm assets
 
-#### Install composer packages
+_Note: you'll need to do this for the common directory and the root directory._
 
-With composer in place, let's get those composer packages:
 ```
-composer install
+cd common
+nvm use
+npm ci
+npm run build
+
+cd ..
+nvm use
+npm ci
+npm run build
 ```
 
-#### Install npm and composer packages in common
+##### Building specific assets
 
-One of our submodules is Tribe Common, which is a shared set of libraries in use by our plugins (see also [Event Tickets](https://github.com/moderntribe/event-tickets)). It uses npm and composer as well, so, let's get those as well.
+Our `npm run build` command is simply an alias to running both our Gulp and webpack builds. You can run them individually like so:
 
 ```
-cd common
-npm install
-composer install
+npm run build:gulp
+npm run build:webpack
 ```
 
-### Using Gulp
-
-Our gulp tasks are documented in our [product-taskmanager README](https://github.com/moderntribe/product-taskmaster#gulp-tasks).
+We also provide `gulp watch` tasks, so feel free to leverage that while tinkering with CSS or JS to run the build process automatically when you change files.
```

Comment:
```suggestion
We also provide `gulp watch` tasks, so feel free to leverage those while tinkering with CSS or JS to run the build process automatically when you change files.
```
"tasks" is plural - so "those" instead of "that"?